src/watchmaker/workers/salt.py

```diff
@@ -40,6 +41,13 @@ class SaltBase(WorkerBase, PlatformManagerBase):
             - *Linux*: ``/srv/watchmaker/salt``
             - *Windows*: ``C:\Watchmaker\Salt\srv``
 
+        salt_content_path: (:obj:`str`)
+            Used in conjunction with the "salt_content" arg.
+            Path to the salt content files inside the provided salt_content
```

Comment:
Would be helpful to note the glob support in the docs...

Reply:
Will do. Missed updating the doc.